guides/source/action_text_overview.md

```diff
@@ -46,6 +46,8 @@ happens after every keystroke, and avoids the need to use execCommand at all.
 ## Installation
 
 Run `rails action_text:install` to add the Yarn package and copy over the necessary migration.
+Also you need to set up Active Storage to use Action Text to manage attachments.
+Please refer to the [Active Storage Overview](https://edgeguides.rubyonrails.org/active_storage_overview.html#transforming-images).
```

Comment:
Users need to know about the whole Active Storage only not transform images. 
I think that the link destination is the top of Active Storage guide is more appropriate.   

Also, you don't need to specify a domain in link during guides. Only `active_storage_overview.html` is enough.

Reply:
Sorry I should have referred to other guide to add link.
Thank you again to suggest.